tests/Unit/Parallel/Test_AlgorithmCore.cpp

```diff
@@ -720,7 +720,6 @@ struct TestMetavariables {
       case Phase::AnyOrderStart:
         return Phase::AnyOrderFinish;
       case Phase::AnyOrderFinish:
```

Comment:
I think there is a `[[fallthrough]]` attribute to mark that this is intentional...